docker-based-rock-ons/netdata_official.rst

```diff
@@ -0,0 +1,106 @@
+.. _netdata_official_rockon:
+
+Netdata (official) Rock-on
+==========================
+
+Before you install the Netdata (official) rock-on, you should understand the
+prerequisites and configurations common to all Rockstor :ref:`rockons_intro`;
+specifically the :ref:`rockons_preinstall` and :ref:`rockons_root` requirements.
+
+
+.. _netdata_official_whatis:
+
+What is Netdata
+---------------
+
+`Netdata <https://www.netdata.cloud/product/>`_ Agent is a free an open-source
+tool to monitor and analyze the performance of your system in real time. It
+gathers a wealth of metrics from the operating system, hardware, as well as
+applications (including rock-ons), and display them in a modern and descriptive
+interface. Netdata also allows you to set different alerts to get notified when
+a specific metric reaches a designated threshold.
+For further details and up-to-date description of Netdata, please visit the
+`project's documentation <https://learn.netdata
+.cloud/docs/overview/what-is-netdata>`_.
+
+.. _netdata_doc:
+
+Netdata Documentation
+---------------------
+
+Netdata's features and possibilities can be greatly expanded and customized to
+better fit your needs, hardware, and preferences. Fortunately, Netdata provides
+and `extensive documentation <https://learn.netdata.cloud/>`_ to learn how to do
```

Comment:
```suggestion
`extensive documentation <https://learn.netdata.cloud/>`_ to learn how to do
```
Extraneous 'and'.

Reply:
Oops... It was meant to be "an", but it actually works better the way you suggested so I'll commit that.